### PR TITLE
README - macOS installation, validating installation

### PR DESCRIPTION
**Pull request by aguynamedben**
The "manual installation" section of the README only has Linux and Windows examples, and I had to dig through issues to figure out how to get rust-bert working on macOS (Apple Silicon).

This adds "macOS + Homebrew" to the manual installation section. Most macOS users building in Rust would consider it fair to call installing pytorch via Homebrew "manual" installation" (trying to avoid pyenv, pip, etc.)

### CODE REVIEW

**Comment by aguynamedben** [comment]
Credit to @knpwrs in https://github.com/guillaume-be/rust-bert/issues/326#issuecomment-1468703653 for providing the actual code 🙏

File: README.md [hunk 1]
```diff
@@ -93,12 +93,29 @@ $Env:LIBTORCH = "X:\path\to\libtorch"
...
+
 ### Automatic installation
 
 Alternatively, you can let the `build` script automatically download the `libtorch` library for you. The `download-libtorch` feature flag needs to be enabled.
 The CPU version of libtorch will be downloaded by default. To download a CUDA version, please set the environment variable `TORCH_CUDA_VERSION` to `cu118`.
 Note that the libtorch library is large (order of several GBs for the CUDA-enabled version) and the first build may therefore take several minutes to complete.
 
+### Verifying installation
+
+Verify your installation (and linking with PyTorch) by cloning the rust-bert source and running an example:
```
Comment (guillaume-be) [inline README.md:111]:
```suggestion
Verify your installation (and linking with libtorch) by adding the `rust-bert` dependency to your `Cargo.toml` or by cloning the rust-bert source and running an example:
```

**Review by guillaume-be — COMMENTED** [review]
Thank you @aguynamedben for adding the instructions for macOS. I have added a small comment since we technically link to libtorch - please let me know what you think

**Comment by aguynamedben** [comment]
@guillaume-be lgtm, thank you for working on this great crate!

**Pull request merged**